test/system/520-checkpoint.bats

```diff
@@ -245,7 +245,7 @@ function teardown() {
     assert "$mac2" == "$mac1" "mac after restore should match"
 
     # restart the container we should get a new ip/mac because they are not static
-    run_podman restart $cid
+    run_podman 0+w restart $cid
```

Comment:
`restart` seems to invoke `stop` which then does the timeout-sigkill thing. I just want to point this out in case it is not 100% common knowledge. Also want to point out that this costs us 10 seconds per restart, so, 40 extra seconds of CI time.

Reply:
Can't we change the default stop-timeout of the containers to 0?

Reply:
I don't know if you mean in the tests, or in `restart`, but either way that's not something I'm confident doing on my own.

Reply:
I didn't investigate it yet but I _guess_ that creating/running the containers with `--stop-timeout` will do the job.

Reply:
I haven't investigated fully either, because that code is scary, but at least some of those containers are `sleep inf`, which is a bad idea all around. Changing to `top` would solve some of these. But again, I'm just not familiar enough with the purpose behind these tests, and don't feel comfortable making that change.